### ISSUE TITLE
Explicitly label closed source apps as such

### ISSUE THREAD

**Issue by ghost** [accrescent/meta#17]
Add "Closed Source" labels to every closed source app tile in all the app sections. This could inform the users better about the apps they use. The label should have full "Closed Source" text in them for extra clarity and awareness

**Comment by matchboxbananasynergy** [comment]
As far as I know, the plan is to instead put an "open-source" label on the open-source apps, and people will be able to filter for that tag so that they're only shown open-source apps if they prefer that.

**Comment by ghost** [comment]
Thanks, also a good idea

**Comment by lberrymage** [comment]
Prerequisite: https://github.com/accrescent/meta/issues/4

Also yes, it seems preferable to do the inverse of this.

**Comment by lberrymage** [comment]
Moving to meta since this also requires devconsole work.

**Comment by lberrymage** [comment]
Closing in favor of #25
